### ISSUE TITLE
Gulp Support

### ISSUE THREAD

**Issue by demisx** [half-pipe/half-pipe#97]
Nice project! However, we've moved away from `grunt` to `gulp` for its simplicity and efficiency. Any chance half-pipe will be `gulp`-based in the nearest future?

**Comment by joefiorini** [comment]
The answer to your question is yes _and_ no. I've already been experimenting with a broccoli-based setup and it's working very well. With broccoli, and I assume the same is true for gulp, there is still a need for a simple task execution tool like grunt to do things like run jshint, tests, etc. So while I don't see us moving completely away from grunt anytime soon, I do see a broccoli-based setup that uses grunt to execute broccoli coming soon.

**Comment by bijmaxx** [comment]
We're in the process of migrating away from sprockets. We're leaning towards gulp over grunt, so a +1 for gulp.

@joefiorini I believe gulp is capable of executing simple tasks like jshint, tests, etc.

**Comment by demisx** [comment]
Yes, `gulp` is capable of doing many things. We are using it heavily for all sort of tasks and, in my opinion, the simplicity of stream driven programming in `gulp` beats Grunt big time for me. I've never looked back.

**Comment by inlikealion** [comment]
Another big **+1** for Gulp. Perhaps a way to provide half-pipe with a parameter for preferred task-runner. That way people could choose the tool they need instead of coupling one with the use of this gem.
